donate/templates/payment/card.html

```diff
@@ -121,11 +121,13 @@ <h2 class="donate-form__actions-heading heading heading--primary">{% trans "Paym
                                 </div>
                             </div>
 
-                            {% if payment_frequency == 'monthly' %}
-                            <footer class="donate-form__footer">
-                                <p>{% trans "I authorize Mozilla Foundation to charge my card in the amount shown below and to continue doing so each month until I cancel." %}</p>
+                            <footer class="donate-form__authorization">
+                                {% if payment_frequency == 'monthly' %}
+                                    <p>{% trans "I authorize Mozilla Foundation to automatically charge my card every month in the amount indicated above, starting today and continuing each month on the same day, until I cancel. I understand that I must cancel at least 5 days before the next scheduled donation by email to <a href='mailto:donate@mozilla.org'>donate@mozilla.org</a>, and that I may only request a refund within 15 days from the date of a donation." %}</p>
```

Comment:
```suggestion
                                    <p>{% trans "I authorize Mozilla Foundation to automatically charge my card every month in the amount indicated above, starting today and continuing each month on the same day, until I cancel. I understand that I must cancel at least 5 days before the next scheduled donation by email to <a href='mailto:{{ email }}'>{{ email }}</a>, and that I may only request a refund within 15 days from the date of a donation." %}</p>
```

Reply:
Let’s move the email into a placeable, to avoid translating twice in case we need donate-thunderbird@ at some point. We will have to switch to a `{% blockquote %}`, though

Reply:
Is this email string already stored somewhere?

Reply:
No, we can do like for the second `blockquote` on this line https://github.com/mozilla/donate-wagtail/blob/2be5d7b63e2085149e0521d2773b4e0aefbf8f07/donate/templates/payment/thank_you_master.html#L15

Reply:
Nevermind, I see how it's implemented.